process_jartic_simple_v2.py

```
@@ -0,0 +1,205 @@
+                            csv_data = pref_zip.read(csv_file).decode('shift_jis', errors='ignore')
+                            lines = csv_data.splitlines()
+                            
+                            # Quick count of valid lines in date range
+                            for line in lines[1:]:  # Skip header
+                                if ',' in line and len(line.split(',')) >= 10:
+                                    # Quick date check on first column
+                                    try:
+                                        time_str = line.split(',')[0]
+                                        if '2024/01' in time_str:  # Quick filter for January 2024
+                                            total_count += 1
+                                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)